package.json

```diff
@@ -19,7 +19,7 @@
     "precommit": "lint-staged",
     "prepublishOnly": "yarn build",
     "test-ci": "lerna run test-ci --stream",
```

Comment:
looks like this is still using lerna. `test-ci` is what ci runs. We should also remove these entries from the other packages.